.travis.yml

```diff
@@ -39,8 +39,11 @@ matrix:
     - name: "make depscheck"
       script: make depscheck
     - name: "make website-lint"
-      script: make website-lint
+      script:
+      - curl -sLo /tmp/terraform.zip https://releases.hashicorp.com/terraform/0.12.18/terraform_0.12.18_linux_amd64.zip
+      - unzip /tmp/terraform.zip -d /tmp
+      - mv /tmp/terraform ~/bin
```

Comment:
we could also move this to the current directory?

Reply:
i tried and it failed, probably due to how terrafmt runs tf